atsign-foundation/noports
# Fix Overflow errors

**- What I did**

- Set the minimum desktop size to 671*467

- Updated the Navigation Rail, Home Screen and Settings screen to ensure it fit properly to the minimum desktop size.

**- How I did it**

**- How to verify it**

1. Run the app
2. Try to minimise the desktop below a size of 671*467, The resizing should be ignored below the minimum size.
3. Navigate to the Terminal, Home and Settings screen and they should be no overflow error in the debug console.

**- Description for the changelog**

Updated to the Terminal Screen, Home Screen and Settings screen to ensure it fit properly to the minimum desktop size.
